src/app/loans/loans-view/transactions-tab/transactions-tab.component.ts

```diff
@@ -90,9 +90,13 @@ export class TransactionsTabComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.hideAccrualsParam = new UntypedFormControl(false);
+    this.hideAccrualsParam = new UntypedFormControl(true);
     this.hideReversedParam = new UntypedFormControl(false);
-    this.setLoanTransactions();
+    if (this.hideAccrualsParam.value) {
+      this.filterTransactions(this.hideReversedParam.value, this.hideAccrualsParam.value);
```

Comment:
Can you double check the two implementations
  hideAccruals() {
    this.filterTransactions(this.hideReversedParam.value, !this.hideAccrualsParam.value);
  }
  
  The original method seems to have an inverted flag while you don't have one in your filtering. One of them would be wrong.